doc/classes/EditorPlugin.xml

```diff
@@ -169,6 +169,7 @@
 			<return type="void">
 			</return>
 			<description>
+				This method is a called automatically when the [EditorPlugin] is disabled through the Project->Project Settings->Plugins interface
```

Comment:
`Called by Godot when the user disables the [EditorPlugin] in the Plugin tab of the project settings window`